feat(kubernetes): example cronjob & container image creation (#13)
adfinis/vault-raft-backup-agent: kubernetes/README.md

> # Cronjob for snapshoting a vault running on Kubernetes

Reviewer: ```suggestion
# Cronjob for snapshotting Vault running on Kubernetes
```

> This assumes kubernetes authentication backend is configured on vault.

Reviewer: ```suggestion
This assumes the Kubernetes authentication backend is configured in Vault.
```

> The container image being used in this cronjob, is authenticating to Vault with the kubernetes authentication backend, with its serviceaccount JWT.

Reviewer: ```suggestion
The script being executed in this cronjob, is authenticating with Vault using the Kubernetes authentication backend, via its serviceaccount JWT.
```

> The role and policy being used must be created before hand and must be used by the cronjob.

Reviewer: You're talking about a Vault role and policy here, not a Kubernetes role right? So there no `role.yaml` missing in the PR.

Author: Yes, it is about Vault role & policy.
